### ISSUE TITLE
Cannot get libraries to load

### ISSUE THREAD

**Issue by deejub44** [ukdtom/ExportTools.bundle#125]
All attempts to get libraries to load fail, Log shows "CRITICAL (core:574) - Exception when calling function 'ValidatePrefs' (most recent call last):". I tried the steps in https://github.com/ukdtom/ExportTools.bundle/issues/103 but that did not help. Log is attached.

[com.plexapp.plugins.ExportTools.log](https://github.com/ukdtom/ExportTools.bundle/files/4644104/com.plexapp.plugins.ExportTools.log)

**Comment by ukdtom** [comment]
It tries to update the file named:

`C:\Users\doug\AppData\Local\Plex Media Server\Plugins\ExportTools.bundle\Contents\\Info.plist'`

Does above file exists on your system, and if so, do the user that PMS is running under have full rights to that file?

**Comment by deejub44** [comment]
I thank you very kindly for the response. There must have been something wrong with my install, because I uninstalled and then reinstalled via WebTools rather than manually, and it now is working. I did read the manual carefully, and I do think I installed it correctly using the manual procedure -- but obviously I screwed something up. I am duly embarassed. Best wishes in these difficult times and stay well.
